hasjob/templates/layout.html

```diff
@@ -36,32 +36,118 @@
+                {%- endfor -%}
+              </select>
+            </div>
+            <div class="filters dropdown col-xs-12 col-sm-4 col-md-3">
+              <div id="job-filters-pay" class="btn-group btn-block no-jshidden">
+                <button type="button" class="btn btn-default dropdown-toggle btn-block" data-toggle="dropdown" aria-expanded="false">
+                  <span id="job-filters-pay-text" class="pay-field"></span>
+                  <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu pay-filter-dropdown" role="menu" aria-labelledby="job-filters-pay">
+                  <li class="clearfix">
+                    <div class="currency-checkbox">
+                      <input type="radio" id="job-filters-na" name="currency" value="NA"></input>
+                      <label for="job-filters-na">Any</label>
+                    </div>
+                    <div class="currency-checkbox">
+                      <input type="radio" id="job-filters-inr" name="currency" value="INR"></input>
+                      <label for="job-filters-inr">INR</label>
+                    </div>
+                    <div class="currency-checkbox">
+                      <input type="radio" id="job-filters-usd" name="currency" value="USD"></input>
+                      <label for="job-filters-usd">USD</label>
+                    </div>
+                    <div class="currency-checkbox">
+                      <input type="radio" id="job-filters-eur" name="currency" value="EUR"></input>
+                      <label for="job-filters-eur">EUR</label>
+                    </div>
+                  </li>
+                  <li class="pay-filter-slider">
+                    <div>
+                      <input type="hidden" name="pmin" id="job-filters-pmin">
+                      <input type="hidden" name="pmax" id="job-filters-pmax">
+                      <div id="pay-slider"></div>
+                    </div>
+                  </li>
+                  <li>
+                    <label for="job-filters-equity" class="equity-label">
+                      <input type="checkbox" name="equity" id="job-filters-equity" {% if request.args.get('equity') %} checked="checked" {% endif %} value="1"></input> Equity</label>
+                  </li>
+                </ul>
+              </div>
+            </div>
+            <div class="filters col-xs-12 col-sm-4 col-md-3">
+              <input id="job-filters-keywords" class="filter-select form-control" type="text" placeholder="Keywords" name="q" value="{{ request.args.q }}"/>
+            </div>
+            <div class="submit-filters col-xs-4 col-sm-2 col-md-2">
+              <button id='job-filters-submit' type="submit" class="btn btn-default submit-btn">
+                <span class="filter-text"><i class="fa fa-filter"></i>Filter</span>
+                <span class="search-text"><i class="fa fa-search"></i>Search</span>
```

Comment:
When JS is disabled, both icons will show. 
One solution is to check in jinja if the view is 'index.html' and if it is, L129 becomes
```html
    <span class="search-text no-jshidden"><i class="fa fa-search"></i>Search</span>
```
Likewise for `filter-text`.